### DISCUSSION TITLE
Paranoia about security with softaculous install.

### DISCUSSION THREAD

**Discussion by biduradhikari** [firefly-iii/firefly-iii#7526] [Firefly III]
Just found out that I can install firefly III with softaculous in my cpanel. HTTPS and 2FA and all in the mix, isn't my financial information out in the wild this way? Is it secure? From the public, from my hosting provider, from anyone else?

I'm just looking for a nice way to enter my transactions on the go (from mobile browser) so that they stay in one place.

**Comment by JC5** [comment]
Does this page help?

https://docs.firefly-iii.org/firefly-iii/more-information/security/
